docs/markdown/Reference-manual.md

```diff
@@ -383,6 +383,10 @@ following.
   `{'NAME1': 'value1', 'NAME2': 'value2'}` or `['NAME1=value1', 'NAME2=value2']`,
   or an [`environment()` object](#environment-object) which allows more
   sophisticated environment juggling.
+- `feed` *(since 0.59.0)*: there are some compilers that can't be told to read
```

Comment:
Perhaps want to write 'programs' rather than 'compilers'?

Reply:
Right. The `capture` description mentions "compilers", so I just went with that. Can change if "programs" is better.